controllers/v1/user-extension.js

```diff
@@ -264,4 +264,70 @@ module.exports = class UserExtension extends Abstract {
     })
   }
 
+
+
+   /**
+  * @api {get} /kendra/api/v1/user-extension/read
```

Comment:
@ankitpws I am not sure if I understood the meaning of this PR what exactly are we trying to do?

Reply:
We are validating in this pr whether the user is a program designer or manager to access block levels data. @aks30